src/screens/AddWalletSheet.tsx

```diff
@@ -145,11 +145,13 @@ export const AddWalletSheet = () => {
                     ) {
                       try {
                         await backupUserDataIntoCloud({ wallets: newWallets });
-                      } catch (e) {
-                        logger.error(e as RainbowError, {
-                          description:
-                            'Updating wallet userdata failed after new account creation',
-                        });
+                      } catch (e: any) {
+                        logger.error(
+                          new RainbowError(
+                            '[AddWalletSheet]: Updating wallet userdata failed after new account creation'
+                          ),
+                          { error: e }
```

Comment:
iirc all these errors have PII in them which is why we're doing the reporting in a weird way here

Reply:
Doesn't that get scrubbed from Sentry anyway?